### ISSUE TITLE
NethVoice report prefix validator doesn't allow for 3 digits prefixes

### ISSUE THREAD

**Issue by Stell0** [NethServer/dev#7043]
**Steps to reproduce**

- when configuring NethVoice 8 from interface, try to specify an international prefix with 1 or 3 (or 4 digits)

**Expected behavior**

Interface should accept the code

**Actual behavior**

Interface gives "Invalid format" error

**Workaround**

- Use a  2 digits prefix 
- change the value of `REPORTS_INTERNATIONAL_PREFIX` variable in module environment file `/home/nethvoiceX/.config/state/environment`
- restart report-api container `systemctl --user restart reports-api`

**Components**

ns8-nethvoice

**See also**

https://community.nethserver.org/t/nethvoice-reports-prefix-for-us/

----

Reported by [Jeff Davis](https://community.nethserver.org/u/Shadowfire)
Also thanks to [Marc](https://community.nethserver.org/u/dnutan) that pointed out the bug in code

**Comment by Amygos** [comment]
## Test Case: International Prefix Configuration in NethVoice 8

**Description**: Verify that the system allows the configuration of international prefixes with 1, 3, or 4 digits in NethVoice 8 after installing the testing version of the module.

### Pre-conditions:
- The testing version of the NethVoice 8 module has been successfully installed.
- The system is up and running.
- The user is logged in to the NethVoice 8 interface with administrative privileges.

### Steps to reproduce:
1. Open the NethVoice 8 interface.
2. Navigate to the configuration section for phone numbers or prefixes.
3. Attempt to specify an international prefix using 1, 3, or 4 digits.
4. Submit the configuration.

### Expected behavior:
- The interface should accept the code without errors.

### Post-conditions:
- None.

**Comment by Amygos** [comment]
Test cases: **KO**
Prefix tested: +1, +254
Error:
```
Validation errors: [reports_international_prefix: Does not match pattern '^(00\d{2}|\+\d{2})$']
```

**Comment by Amygos** [comment]
Repeat the above test cases

**Comment by andrea-marchionni** [comment]
Test case OK
Prefix tested: +1, +10, +11, +100, +200, +254

**Comment by Amygos** [comment]
Released [1.0.2](https://github.com/nethesis/ns8-nethvoice/releases/tag/1.0.2)
